### DISCUSSION TITLE
Webhooks for realtime presence similar to pusher

### DISCUSSION THREAD

**Discussion by karandwivedi42** [supabase/supabase#13505] [Feature Requests]
Can we add webhooks which can be triggerred with presence event data similar to Pusher (https://pusher.com/docs/channels/server_api/webhooks/#presence-events)

**Comment by pierroo** [comment]
Did you find an answer to this question @karandwivedi42 ?

**Reply by karandwivedi42** [reply]
Sorry, no. Ended up using pusher
